docs/userGuide/usingPlugins.md

```diff
@@ -35,17 +41,42 @@ For example:
   ],
   "pluginsContext": {
     "plugin1": {
-    	"input": "Input for Plugin 1"
+      "input": "Input for Plugin 1"
     },
     "plugin2": {
-    	"data": "Data for Plugin 2"
-    },
+      "data": "Data for Plugin 2"
+    }
   }
 }
 ```
 
+## Using Built-in Plugins
+
+
+MarkBind has a set of built-in plugins that can be used immediately without installation.
+
+<include src="plugins/algolia.mbdf" />
+<include src="plugins/codeBlockCopyButtons.mbdf" />
+<include src="plugins/tags.mbdf" />
+<include src="plugins/googleAnalytics.mbdf" />
+
+## Using External Plugins
+
+### Adding External Plugins
+
+<tip-box type="warning">
+
+**WARNING:** Plugins are executable programs that can be written by anyone. This means that they might contain malicious code that may damage your computer.
+
+Only run plugins from sources that you trust. Do not run the plugin if the source/origin of the plugin cannot be ascertained.
+</tip-box>
+
+Plugins come as `.js` files. To install an external plugin, simply put it in the `_markbind/plugins` folder. To use the plugin, update the `site.json` file the same way you did for built-in plugins:
```

Comment:
is there supposed to be content after the `:`?

Reply:
No. Will remove.